### DISCUSSION TITLE
Client-ca used for

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#6490] [Q&A]
Experts. Our use case we are not planning to generate user certificate from Kafka rather will use our bank provided pki certificate (I.e type: tls-external during user creation  ) Which allows a general renewal and standards for everyone in company.  Also listener are configured to use our own certificate too. 

want to know  what is the importance of client-ca in our use case  context ? 

If I understood reading some where,  client-ca be used only for ssl handshake in this case by using  the public key? 

is this true understanding  ?  

Where else the client-ca be used ?

**Comment by scholzj** [comment]
There are docs for providing your own Clients CA. If you do not plan to use the User Operator to issue the user certificates, you can just provide some _dummy_ private key (create the Secret and the record inside, but use some PLACEHOLDER value instead of the real private key).

**Reply by sreejesh123** [reply]
Thank you, just to confirm then Is client ca mainly used only for issuing user certificate?

**Reply by scholzj** [reply]
The Clients CA public key is used to establish trust between the user and the broker. The private key is needed only for singing the user certificates.
